### PR TITLE
feat(caip): add mt pelerin asset map

### PR DESCRIPTION
**Pull request by stackedq**

### CODE REVIEW

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  [btcAssetId]: 'BTC',
+  'eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f': 'DAI',
+  'eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7': 'USDT',
+  'eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48': 'USDC',
+  'eip155:1/erc20:0x0f17bc9a994b87b5225cfb6a2cd4d667adb4f20b': 'jEUR',
+  'eip155:1/erc20:0x53dfea0a8cc2a2a2e425e1c174bc162999723ea0': 'jCHF',
+  'eip155:1/erc20:0x7409856cae628f5d578b285b45669b36e7005283': 'jGBP',
+  'eip155:1/erc20:0x2260fac5e5542a773aa44fbcfedf7c193bc2c599': 'WBTC',
+  // AVAX
+  [avalancheAssetId]: 'AVAX',
```
Comment (stackedq) [inline packages/caip/src/adapters/mtpelerin/index.ts:31]: AVAX on MtPelerin is on avalanche-network, I'm not sure if we can support that.

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  // FANTOM
+  'eip155:1/erc20:0x4e15361fd6b4bb609fa63c81a2be19d873717870': 'FTM',
```
Comment (stackedq) [inline packages/caip/src/adapters/mtpelerin/index.ts:33]: Same as Fantom here, it's on `fantom_mainnet`.

**Review by gomesalexandre — CHANGES_REQUESTED** [review]
See comments - this will only be able to represent an asset on one chain, whereas Mt Pelerin has an intersection of assets we support on both C-Chain and Mainnet

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    '[fantom_mainnet]/0x846e4D51d7E2043C1a87E0Ab7490B93FB940357b': 'UST',
+    '[rsk_mainnet]/0x0000000000000000000000000000000000000000': 'RBTC',
+    '[rsk_mainnet]/0x2d919f19d4892381d58edebeca66d5642cef1a1f': 'RDOC',
+    '[rsk_mainnet]/0xef213441a85df4d7acbdae0cf78004e1e486bb96': 'USDT',
+    '[rsk_mainnet]/0x2acc95758f8b5f583470ba265eb685a8f45fc9d5': 'RIF',
+    '[ethChainId]/0x89d71DfbDD6ebeCd0dfe27D55189F903169d2991': 'THDX'
+* }
+*/
+
+export const mtPelerinSymbolToAssetIds = (id: string): string[] => MtPelerinSymbolToAssetIds[id]
```
Comment (0xdef1cafe) [inline packages/caip/src/adapters/mtpelerin/index.ts:102]:
```suggestion
export const mtPelerinSymbolToAssetIds = (id: string): AssetId[] => MtPelerinSymbolToAssetIds[id]
```

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+export const assetIdToMtPelerinSymbol = (assetId: string): string | undefined => {
```
Comment (0xdef1cafe) [inline packages/caip/src/adapters/mtpelerin/index.ts:104]:
```suggestion
export const assetIdToMtPelerinSymbol = (assetId: AssetId): string | undefined => {
```

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+ * since some MtPelerin assets could be on multiple chains and their default
+ * chain won't be exactly the same as ours.
+ *
+ * https://developers.mtPelerin.com/integration-guides/options -> "net"
+ * source of truth per MtPelerin
+ */
+const chainIdToMtPelerinNetworkCodeMap: Record<ChainId, string> = {
+  [ethChainId]: 'mainnet',
+  [btcChainId]: 'bitcoin_mainnet',
+  [avalancheChainId]: 'avalanche_mainnet',
```
Comment (gomesalexandre) [inline packages/caip/src/adapters/mtpelerin/index.ts:120]: Comment that this is actually the C-Chain?

**Review by 0xdef1cafe — DISMISSED** [review]
type sig nit picks

**Review by gomesalexandre — APPROVED** [review]
⛰️

**Pull request merged**